### ISSUE TITLE
Screening criteria and implications for mental health

### ISSUE THREAD

**Issue by hanjiaxu** [uchicago-computation-workshop/marc_berman#8]
The idea brought up in the paper to measure cognitive effort is fascinating. My first question is that in VWMT dataset, healthy control was screened with a clinical depression diagnosis, so why clinical depression was specifically screened in this dataset? It seems that MTAS didn't screen dipression particularly, and SART and TMT dataset just screened mental status in general.

I'm also wondering if there are any follow up research in this area that looks up the cognitive efforts among patients with depression and anxiety? The paper mentioned that fractal scaling is suppressed under high anxiety and low fatigue in breast cancer patients, has research studies looked whether depression has a similar effect for brain function? In the resting state, the fractal scaling is also suppressed among anxiety patients?

**Comment by bermanm** [comment]
Yes, I am going to talk about this tomorrow.  For the VWMT dataset this was a control group, so we wanted that group to free of any psychopathology, but I will discuss depression tomorrow a bit.

**Comment by Fangfangwan** [comment]
Thank you for bringing our the question that I'm also confused about.
